app/controllers/projects_controller.rb

```diff
@@ -5,5 +5,24 @@ def index
     end
     def show 
         @project = Project.find(params[:id])
+
+        #make last name array accessible to form for user addition from UI
+        @user_last_names = ["<Select User>"]
```

Comment:
This should use the `include_blank` option on the rails view instead: https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/select